### PR TITLE
Improve User Guide to fix PED issues

### LINKED ISSUE #171: [PE-D] Session description should reflect the function better
Instead of saying those that have not been called, should instead write "Displays the full list of people and resets their call status to 'not called' "

-------------
Labels: `severity.Low` `type.DocumentationBug`
original: clementkfj/ped#6

### PR DESCRIPTION
**Pull request by lrnshk**
Fixes #171, #199, #170, #177, #166, #157, #182, #167, #168

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -196,12 +196,55 @@ Deletes all persons with SHN periods that are completed at time of command call
...
+Under Call Status for each person, there are two components:
+* Call status indicator
+  * Represents whether a person has been called in the current SHN enforcement session.
+  * Displayed as 'Called:' followed by a red cross or green tick.
+  * Red cross indicates that a person has not been called for the current SHN enforcement session.
+  * Green tick indicates that a person has been called for the current SHN enforcement session, regardless of whether the call was successful.
+* Non-compliance counter 
+  * Represents the number of failed attempts that have been made to call the person 
+  * Displayed as 'Failed: X time(s)' where X is the number of failed contact attempts 
+  * The number of failed contact attempts recorded is cumulative over the entire period of Track2Gather usage. It does not reset to zero when a new SHN enforcement session is created.
```
Comment (frederickpek) [inline docs/UserGuide.md:223]: Perhaps "contact" can be replaced with "call".

**Review by frederickpek — CHANGES_REQUESTED** [review]
Minor changess

**Review by frederickpek — APPROVED** [review]
Looks good to merge!

File: docs/UserGuide.md [hunk 2]
```diff
@@ -196,12 +196,55 @@ Deletes all persons with SHN periods that are completed at time of command call
...
-### Showing call schedule : `schedule`
+### SHN enforcement mode
+SHN enforcement mode contains a special group of features that enables users to track the call statuses of persons in Track2Gather. 
+The purpose of this mode is to facilitate the calling of persons to enforce their SHN. 
+
+The intended use of this mode is to:
+* create a new calling session every day (henceforth referred to as 'SHN enforcement session').
+* mark successful/failed attempts to contact the persons in order to track their behaviour throughout their SHN.
+
+In SHN enforcement mode, all people who have been called for the current SHN enforcement session will be *filtered away from the schedule and thus the display i.e. they will not show up on the interface.*
```
Comment (wei2912) [inline docs/UserGuide.md:207]: Think this line can be simplified, e.g. "filtered out of the schedule and be hidden on the interface" perhaps? Can't really think of a better phrasing now tbh

File: docs/UserGuide.md [hunk 3]
```diff
@@ -196,12 +196,55 @@ Deletes all persons with SHN periods that are completed at time of command call
...
+
+There are 4 features relevant to this mode: `session`, `schedule`, `scall`, and `fcall`.
+* `session` and `schedule` activate SHN enforcement mode.
+* `scall` and `fcall` are intended to work best in SHN enforcement mode, but can be used outside of enforcement mode.
+
+#### How the information is reflected on the interface
+Under Call Status for each person, there are two components:
```
Comment (wei2912) [inline docs/UserGuide.md:214]: `Call Status` can be enclosed in quotes to indicate that it is a heading in the UI.

File: docs/UserGuide.md [hunk 4]
```diff
@@ -211,11 +254,15 @@ Format: `scall INDEX`
 * Updates the person at the specified `INDEX` as called.
 * The index **must be a positive integer** (e.g. 1, 2, 3, ..)
 * The index **must not exceed the total number of persons** in the contacts list
-* If `session` or `schedule` was previously called, the person at the specified `INDEX` will be removed from the list.
+* If Track2Gather is in enforcement mode, the person at the specified index will be removed from the schedule and thus the display.
```
Comment (wei2912) [inline docs/UserGuide.md:257]: It might be good to add the point on the call status being updated here, as a subpoint.

File: docs/UserGuide.md [hunk 5]
```diff
@@ -225,19 +272,18 @@ Format: `fcall INDEX`
 * Updates the person at the specified `INDEX` as called, and increments the person's non-compliance counter by 1.
 * The index **must be a positive integer** (e.g. 1, 2, 3, ..)
 * The index **must not exceed the total number of persons** in the contacts list
-* If `session` or `schedule` was previously called, the person at the specified `INDEX` will be removed from the list.
+* If Track2Gather is in SHN enforcement mode, the person at the specified index will be removed from the schedule and thus the display.
```
Comment (wei2912) [inline docs/UserGuide.md:275]: Same as previous comment.

**Review by wei2912 — CHANGES_REQUESTED** [review]
Have a few minor changes, otherwise LGTM!

**Review by wei2912 — APPROVED** [review]
LGTM!

**Pull request merged**